docs/java/workflows.md

```diff
@@ -200,7 +200,7 @@ They shouldn't use any constructs that rely on system time.
   - Use `WorkflowQueue` instead of `BlockingQueue`.
 - Use `Workflow.getVersion` when making any changes to the Workflow code. Without this, any deployment of updated Workflow code
   might break already running Workflows.
-- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Workflow execution path.
+- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Worklfow Execution path.
```

Comment:
```suggestion
- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Workflow Execution path.
```